frontend/src/components/Chat/DirectMessageSelect.tsx

```diff
@@ -17,17 +16,27 @@ export default function DirectMessageSelect({
 }: DirectMessageSelectProps): JSX.Element {
   const { myPlayerID, players, directMessageChains } = useCoveyAppState();
   const [chosenDirectID, setDirectID] = useState<string>('');
-  const playersWithChats: Player[] = [];
-  const playersWithoutChats: Player[] = [];
 
-  players.forEach(player => {
-    const directMessageId = [myPlayerID, player.id].sort().join(':');
-    if (directMessageChains[directMessageId]) {
-      playersWithChats.push(player);
-    } else if (player.id !== myPlayerID) {
-      playersWithoutChats.push(player);
+  const chattedWithPlayers: string[] = [];
+  const playersWithChats: DirectMessageParticipant[] = [];
+
+  Object.values(directMessageChains).forEach(value => {
+    if (value.participants) {

```

Comment:
Maybe add a comment here explaining that "participants" means that a chain has already started? I feel like this logic isn't intuitive to read if you don't know that particular part of our logic flow